Client/multiplayer_sa/CMultiplayerSA.cpp

```diff
@@ -578,6 +578,76 @@ CMultiplayerSA::CMultiplayerSA()
     m_dwLastStaticAnimID = eAnimID::ANIM_ID_WALK;
 }
 
+namespace GrainEffect
+{
+
+static BYTE ucGrainEnabled = FALSE;
+static DWORD dwGrainStrength = 1.0f;
```

Comment:
Should be `bool` and `float`. BYTE and DWORD are really just legacy stuff

Reply:
bool and BYTE is not the same. A compiler does not guarantee equality sizeof(bool) and sizeof(BYTE). This is not an equitable type conversion. The more correct would be unsigned char instead of BYTE. The floating point value is a misprint, it should be unsigned int actually.